deeplay/external/external.py

```diff
@@ -100,11 +101,24 @@ def build(self) -> nn.Module:
         obj = self.classtype(*args, **kwargs)
 
         self._run_hooks("after_build", obj)
-
         return obj
 
     create = build
 
+    def __construct__(self):
+        with not_top_level(ExtendedConstructorMeta):
+            self._modules.clear()
+            self._is_constructing = True
+
+            kwargs = self.kwargs.copy()
+            # hack for external
+            classtype = kwargs.pop("classtype")
+
+            self.__init__(*(classtype, self._args), **kwargs)
+            self._run_hooks("after_init")
+            self._is_constructing = False
+            self.__post_init__()
+
```

Comment:
Should probably not redefine the entire __construct__ method. Maybe we can abstract it into a separate `args, kwargs = self.get_init_arguments()` which can be different for DeeplayModule and External

Reply:
I agree. https://github.com/DeepTrackAI/deeplay/pull/60/commits/4141508bca7a59d1ac07075d9a7ea9bb44154448 corrects this!